src/pmod_maxsonar.erl

```diff
@@ -54,6 +54,14 @@ handle_info({Port, {data, Data}}, #state{port = Port} = State) ->
             % Val is given in inches
             Val = (D1 - $0) * 100 + (D2 - $0) * 10 + (D3 - $0),
             {noreply, State#state{last_val = Val}};
+        % All subsequent calls to pmod_maxsonar:get/0 will return
```

Comment:
If you change the comment here we can merge this: 

"Sometimes for no obvious reason we receive a different value from the sonar.  
Instead of $R we get two garbage characters"